README.md

```diff
@@ -76,7 +76,7 @@ led [green|red] [mode]                    sets the led mode
     [lantern|stpatricks|easter|labourday]
     [eid|dragonboat|independenceday|onam]
     [diwali|thanksgiving|christmas]
-    [heavymetal|dance|kecak|random]
+    [heavymetal|dance|kecak|sandstorm|random]
```

Comment:
This line extends into the 'section' for the descriptions. Maybe you can move random to a new line?